typescript/graphql-nestjs/src/main.ts

```diff
@@ -3,6 +3,7 @@ import { AppModule } from './app.module'
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule)
+  app.enableShutdownHooks()
```

Comment:
FWIW, this is strictly speaking optional too since we don't actually have any lifecycle hooks in this code. Still nice to have for future-proofing IMO.